vms/vms/settings.py

```diff
@@ -61,8 +61,8 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'vms',
-        'USER': 'myuser',
-        'PASSWORD': 'mypassword',
+        'USER': 'vmsadmin',
+        'PASSWORD': '0xdeadbeef',
         'HOST': 'localhost',
```

Comment:
> Yes.. syncdb didn't work until I fixed settings.py as part of
> installation.  Based on your reply, we should keep this change.

That is expected. You need to replace the username and password by your own. It cannot be general for everyone. So the previous wording is fine. Revert this change.

> Also, there were many steps where we got stuck at installation. I thought
> someone fixed the documentation that day. If not I will contribute to that
> as well.

I saw lot of great fixes in documentation. Please go ahead and raise issues or fix the issues present.

> Also is there a repo of the expectations of the VMS web app  ? I.e. how do
> we want to store volunteers, organizations, design of the page, what kind
> of data each page should consume etc. ?

[This](https://github.com/systers/vms/tree/develop/docs) has all the docs. Go through them and let us know if you still don't find what you are looking for.